HeliPort/LoginItemManager.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  LoginItemManager.swift
+//  HeliPort
+//
+//  Created by SS on 7/14/20.
+//  Copyright © 2020 OpenIntelWireless. All rights reserved.
+//
+
+import Foundation
+import ServiceManagement
+
+class LoginItemManager {
+
+    private static let launcherId = Bundle.main.bundleIdentifier! + "-Launcher"
+
+    public class func isEnabled() -> Bool {
+
+        guard let jobs =
```

Comment:
I think there might be a simpler way to do this that I saw in a different project https://github.com/MSzturc/ThinkpadAssistant/blob/master/ThinkpadAssistant/AppDelegate.swift#L62-L66

Reply:
Using your method requires the launcher to be running all the time to serve as an indicator of the App being auto launched, it's possible that HeliPort will report a wrong status if both HeliPort and its launcher were terminated.

The method I used reads directly from `launchd`, but Apple decided to deprecate this API many years ago.

Is it better to use the presence of a running app as an indicator? Or maybe we should avoid using deprecated APIs?